docs/en/sql-reference/sql-functions/map-functions/map_from_entries.md

```diff
@@ -0,0 +1,42 @@
+---
+displayed_sidebar: docs
+---
+
+# map_from_entries
+
+
+
+Creates a MAP value from the given array of entries that contains just 2 fields.
+
+This function is supported from v3.4 onwards.
+
+## Syntax
+
+```Haskell
+MAP map_from_entries(ARRAY entries)
+```
+
+## Parameters
+
+- `entries`: array of structs. Make sure the structs just have two fields.
+
+## Return value
+
+Returns a MAP that is constructed from the structs.
```

Comment:
need to describe the expected behavior when there are duplicated keys in array

Reply:
Done